src/test/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateServiceTest.java

```diff
@@ -112,6 +113,7 @@ void setUp() {
     @DisplayName("When caseData with a hearing that matches one from hearingGetResponse is given,"
             + "updateHearing updates the correct hearing")
     @Test
+    @Disabled
```

Comment:
these tests are no longer valid? have they effectively been moved to another class? can we remove them?

Reply:
removed it